deepchecks/nlp/utils/text_properties_models.py

```diff
@@ -0,0 +1,221 @@
+import importlib
+import pathlib
+from functools import lru_cache
+from typing import Optional, Union
+
+import requests
+from nltk import corpus
+
+MODELS_STORAGE = pathlib.Path(__file__).absolute().parent / '.nlp-models'
+
+
+def get_create_model_storage(models_storage: Union[pathlib.Path, str, None] = None):
```

Comment:
what happens here if you use cache=true?

Reply:
In that case `get_create_model_storage()` will be called once and store the model to disk, from that time one - the model will be fetched from the cache, but after restart - the `model storage` benefits will come to hand as we won't need to download the model for the net, but only load it from disk

Reply:
not critical but would describe this flow in some docstring

Reply:
I prefer not to write it in this method, as this can be changed in the future and then we will left with unwanted comment :-)